.travis.yml

```diff
@@ -55,12 +70,14 @@ cache:
 env:
   global:
     - TASK_TESTS=1
+    - TASK_STAN=0
     - TASK_DOCS=0
     - PIMCORE_ENVIRONMENT=test
     - PIMCORE_TEST=1
     - PIMCORE_TEST_URL=http://pimcore-test.dev
     - PIMCORE_TEST_DB_DSN="mysql://root@localhost/pimcore_test"
     - PIMCORE_TEST_CACHE_REDIS_DATABASE=1
+    - PIMCORE_TEST_CACHE_REDIS_DATABASE=1
```

Comment:
This is the same as the line above.

Reply:
yes, but that was the case before as well. I will wait for #4435 and continue working on this

Reply:
it was not, sorry. But this is WIP anyway :)